### PR TITLE
RBL: add interface GD32F103CBT6

### PR DESCRIPTION
**Pull request by zoujixing**

### CODE REVIEW

**Comment by rsalveti** [comment]
I took this pull request as reference and updated the blenano2 firmware (using the GD32F103CBT6 interface) to the current DAPLink head, you can check my current branch at https://github.com/OpenSourceFoundries/DAPLink/tree/blenano2-osf. Tested my branch with difference use cases (storage, uart and flashing), and it all seems to work quite well.

Let me know if you want to update this pull request based on my branch, or if you would like me to just propose that instead.

File: source/hic_hal/GigaDevice/GD32F103CBT6/IO_Config.h [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+#include "gd32f10x.h"
+#include "compiler.h"
+#include "daplink.h"
+
+COMPILER_ASSERT(DAPLINK_HIC_ID == DAPLINK_HIC_ID_GD32F103CB);
+
+//USB control pin
+#define USB_CONNECT_PORT_ENABLE()    RCC_APB2PeriphClock_Enable(RCC_APB2PERIPH_GPIOA, ENABLE)
+#define USB_CONNECT_PORT_DISABLE()   RCC_APB2PeriphClock_Enable(RCC_APB2PERIPH_GPIOA, DISABLE)
+#define USB_CONNECT_PORT             GPIOA
```
Comment (findlayfeng) [inline source/hic_hal/GigaDevice/GD32F103CBT6/IO_Config.h]: why? 
View the manual. This pin has nothing to do with usb.

**Comment by brianesquilona** [comment]
#257 was merged for stm32f103xb hic implementation.

**Pull request closed without merging**